test/powershell/Modules/Microsoft.PowerShell.Utility/Update-TypeData.Tests.ps1

```diff
@@ -143,102 +143,102 @@ Describe "Update-TypeData basic functionality" -Tags "CI" {
-        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should be "ValueShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
+        $ps.HadErrors  |Should -BeTrue
+        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should -BeExactly "ValueShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
 	}
 
 	It "Update-TypeData with Invalid DynamicType with No MemberName should throw Exception"{
 	    $null = $ps.AddScript('Update-TypeData -MemberType NoteProperty -Value "Error" -TypeName System.String')
         $ps.Invoke()
-        $ps.HadErrors  |Should be $true
-        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should be "MemberNameShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
+        $ps.HadErrors  |Should -BeTrue
+        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should -BeExactly "MemberNameShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
 	}
 
 	It "Update-TypeData with Invalid DynamicType with No Value should throw Exception"{
 		$null = $ps.AddScript('Update-TypeData -MemberType NoteProperty -MemberName TestNote -TypeName System.String')
         $ps.Invoke()
-        $ps.HadErrors  |Should be $true
-        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should be "ValueShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
+        $ps.HadErrors  |Should -BeTrue
+        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should -BeExactly "ValueShouldBeSpecified,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
 	}
 
 	It "Update-TypeData with Invalid DynamicType with Empty TypeData should throw Exception"{
 		$null = $ps.AddScript("Update-TypeData -TypeName System.String")
         $ps.Invoke()
-        $ps.HadErrors  |Should be $true
-        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should be "TypeDataEmpty,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
+        $ps.HadErrors  |Should -BeTrue
+        $ps.Streams.Error[0].FullyQualifiedErrorId  | Should -BeExactly "TypeDataEmpty,Microsoft.PowerShell.Commands.UpdateTypeDataCommand"
 	}
 
 	It "Update-TypeData with Valid Standard Members Serialization Method String should work"{
         $null = $ps.AddScript('Update-TypeData -SerializationMethod string -StringSerializationSource Length -TargetTypeForDeserialization string -TypeName System.String')
         $ps.Invoke()
         $ps.Commands.Clear(); $null = $ps.AddScript('"string".PSStandardMembers.SerializationMethod')
-        $ps.Invoke()| Should Be "String"
+        $ps.Invoke()| Should -BeExactly "String"
         $ps.Commands.Clear(); $null = $ps.AddScript('"string".PSStandardMembers.StringSerializationSource')
-        $ps.Invoke()| Should Be 6
+        $ps.Invoke()| Should -Be 6
         $ps.Commands.Clear(); $null = $ps.AddScript('"string".PSStandardMembers.TargetTypeForDeserialization')
-        $ps.Invoke()| Should Be "string"
+        $ps.Invoke()| Should -BeExactly "string"
```

Comment:
Add whitespace between Invoke() and pipe.  Apply to rest of this file (I think you can just search/replace and verify).